app/Resources/assets/js/admin.js

```diff
@@ -0,0 +1,69 @@
+var $ = require('jquery');
+window.$ = $;
+window.jQuery = $;
```

Comment:
You shouldn't need either of these unless we still have some JavaScript code in a template somewhere that is relying on these to be globally available. The only place I could see was `login.html.twig`. We could move that little bit of JavaScript here, or into a new `login` entry as an example of a page-specific JS file.